components/form/pta/src/index.js

```diff
@@ -5,19 +5,36 @@ import {paramsToQueryString} from './querystring'
 const BASE_CLASS = 'sui-FormPta'
 const CONTENT_CLASS = `${BASE_CLASS}-content`
 const REMOVE_DRAFT = 'REMOVE_DRAFT'
+const SUBMIT_SUCCESS_EVENT_TYPE = 'SUBMIT_FORM_SUCCEEDED'
+const SUBMIT_ERROR_EVENT_TYPE = 'SUBMIT_FORM_FAILED'
+const MESSAGE_EVENT_TYPE = 'message'
 
 class FormPta extends Component {
   constructor(props) {
     super(props)
 
-    const {formUrl: BASE_URL, ...settings} = this.props
+    const {formUrl: BASE_URL, onSubmit, onError, ...settings} = this.props
     const QUERY = paramsToQueryString(settings)
     const formUrl = `${BASE_URL}?${QUERY}`
 
+    this.handleMessage = ({data: {type}}) => {
+      type === SUBMIT_SUCCESS_EVENT_TYPE && onSubmit && onSubmit()
```

Comment:
That could work as well :D I usually prefer to avoid this and just give a default value for the prop as an empty func, but this would do the work.

Reply:
👍